lib/mix/tasks/transport/validate_data.ex

```diff
@@ -19,16 +19,16 @@ defmodule Mix.Tasks.Transport.ValidateData do
     |> Mongo.find("datasets", %{}, pool: @pool)
     |> Enum.map(&Dataset.new/1)
     |> Enum.each(fn(dataset) ->
-      %{project: project, name: dataset.slug, url: dataset.download_uri}
+      %{project: project, name: dataset.id, url: dataset.download_uri}
       |> DataValidation.create_feed_source
       |> case do
         {:ok, feed_source} ->
           :ok = DataValidation.validate_feed_source(%{project: project, feed_source: feed_source})
           Logger.info(" <message>  Validating dataset")
-          Logger.info(" <slug>     #{dataset.slug}")
+          Logger.info(" <id>     #{dataset.id}")
```

Comment:
For logging purposes, do we want the `id` or the `slug` ?

Reply:
BOTH! ;) I’m changing it for all